frontend/src/metabase/collections/components/BaseTableItem.jsx

```diff
@@ -49,12 +48,10 @@ export function BaseTableItem({
   isSelected,
   isPinned,
   linkProps = {},
-  hasBottomBorder = true,
```

Comment:
Not sure why this unused var isn't caught by our linter since this is a javascript file